app/crud.py

```diff
@@ -0,0 +1,35 @@
+from sqlalchemy.orm import Session
+from . import models, schemas
+
+def get_city(db: Session, city_id: int):
+    return db.query(models.City).filter(models.City.id == city_id).first()
+
+def get_cities(db: Session):
+    return db.query(models.City).all()
+
+def create_city(db: Session, city: schemas.CityCreate):
+    db_city = models.City(name=city.name, additional_info=city.additional_info)
+    db.add(db_city)
+    db.commit()
+    db.refresh(db_city)
+    return db_city
+
+def delete_city(db: Session, city_id: int):
+    db_city = db.query(models.City).filter(models.City.id == city_id).first()
+    if db_city:
+        db.delete(db_city)
+        db.commit()
+    return db_city

```

Comment:
Similar to `get_city`, consider handling the case where the city with the given `city_id` does not exist before attempting to delete. This will prevent unnecessary database operations and potential errors.